packages/terra-form-select/CHANGELOG.md

```diff
@@ -4,10 +4,16 @@ ChangeLog
 Unreleased
 ----------
 ### Added
+* Added Combobox and ComboboxField exports and doc site entries
+* Added DropdownSelect and DropdownSelectField exports and doc site entries
```

Comment:
What is the DropdownSelect? How is this different from the Default Select?

Reply:
DropdownSelect is the DefaultSelect, just renamed. See: https://github.com/cerner/terra-core/pull/2549#issuecomment-519890741

![image](https://user-images.githubusercontent.com/34115417/62895272-250b3d80-bd14-11e9-93b5-ac32aea7e03e.png)

Reply:
We decided against naming DefaultSelect ➡️Select due to passivity (in case people are pulling in Select via `import Select from 'terra-form-select/lib/Select';`, which is common enough). We want the default export to remain `lib/Select.js`, which now points to the switch selector between the actual individual variant implementations.

Reply:
Hmm, that's tricky. I would expect the default export to be `Select`. A `SingleSelect` might be another option to consider if it absolutely must have a different name than `Select`.

Reply:
It's definitely not ideal, but passivity is a virtue/feature.

I can vibe with `SingleSelect`.